source/install/config-ssl-http2-nginx.rst

```diff
@@ -0,0 +1,97 @@
+.. _config-ssl-http2-nginx:
+
+Configuring NGINX with SSL and HTTP/2
+=====================================
+
+Using SSL gives greater security by ensuring that communications between Mattermost clients and the Mattermost server are encrypted. It also allows you to configure NGINX to use the HTTP/2 protocol.
+
+Although you can configure HTTP/2 without SSL, both Firefox and Chrome browsers support HTTP/2 on secure connections only.
+
+You can use any certificate that you want, but these instructions show you how to download and install certificates from *Let's Encrypt*.
+
+**To SSL and HTTP/2:**
+
+1. Log into the server that hosts NGINX and open a terminal window.
+2. Install git.
+
+  - If you are using Ubuntu or Debian:  ``sudo apt-get install git``
+  - If you are using RHEL: ``sudo yum install git``
+
+3. Clone the Let's Encrypt repository on GitHub.
+
+  ``git clone https://github.com/letsencrypt/letsencrypt``
+
+4. Change to the ``letsencrypt`` directory.
+
+  ``cd letsencrypt``
+
+5. Stop NGINX.
+
+  On Ubuntu 14.04 and RHEL 6.6:
+  
+  ``sudo service nginx stop``
+  
+  On Ubuntu 16.04 and RHEL 7.1:
+  
+  ``sudo systemctl stop nginx``
+
+6. Run ``netstat`` to make sure that nothing is listening on port 80.
+
+  ``netstat -na | grep ':80.*LISTEN'``
+
+7. Run the Let's Encrypt installer.
+
+  ``./letsencrypt-auto certonly --standalone``
+
+  When prompted, enter your domain name. The certificate is located in  ``/etc/letsencrypt/live``
+
+8. Open the file ``/etc/nginx/sites-available/mattermost`` as root in a text editor and update the *server* section to incorporate the highlighted lines in the following sample. Make sure to replace {domain-name} with your own domain name, in 2 places.
```

Comment:
The text says to put it into the server section, and the sample has the hightlighted lines within the server section, so it should be ok.

Actually, there's a couple of lines that are not highlighted. I want to leave out as much of the existing config file as possible in order to help future-proof this topic. th